networkx/algorithms/euler.py

```diff
@@ -253,6 +274,20 @@ def has_eulerian_path(G, source=None):
     -------
     Bool : True if G has an Eulerian path.
 
+    Example
```

Comment:
If you do end up accepting the wording suggestion for the other example, similar wording should be applied here for consistency.